pkg/importer/data-processor_test.go

```diff
@@ -389,8 +389,8 @@ var _ = Describe("Resize", func() {
 	})
 
 	It("Should return same value as replaced function", func() {
-		replaceAvailableSpaceBlockFunc(func(dataDir string) int64 {
-			return int64(100000)
+		replaceAvailableSpaceBlockFunc(func(dataDir string) (int64, error) {
+			return int64(100000), nil
 		}, func() {
```

Comment:
We should add a test where the AvailableSpaceBlockFunc returns an error.

Reply:
So in this particular case we ignore errors from the function and log them. Then continue on base on the result. So adding the test would not add much value.